### PR TITLE
[STATS] Fix native coin holders growth chart

### PR DESCRIPTION
**Pull request by sevenzing**
impl https://github.com/blockscout/blockscout-rs/issues/770

### CODE REVIEW

File: stats/stats/src/charts/lines/native_coin_holders_growth.rs [hunk 1]
```diff
@@ -72,10 +64,296 @@ impl crate::Chart for NativeCoinHoldersGrowth {
...
+                    .all(blockscout)
+                    .await?;
+                let n = rows.len() as u64;
+                all_rows.extend(rows);
+                if n < limit {
+                    break;
+                }
+            }
+            all_rows
+        };
```
[the remark this answers is not in the document]
Comment (sevenzing) [reply stats/stats/src/charts/lines/native_coin_holders_growth.rs:218]: yeah, maybe

File: stats/justfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ test *args:
     cargo test {{args}} -- --include-ignored
 
 test-with-db *args:
-    just db-port="{{test-db-port}}" db-name="" docker-name="{{docker-name}}-test" start-postgres
+    -just db-port="{{test-db-port}}" db-name="" docker-name="{{docker-name}}-test" start-postgres
```
Comment (rimrakhimov) [inline stats/justfile:28]: Should `-` not be here?

**Review by rimrakhimov — APPROVED** [review]

File: stats/justfile (hunk 2 above)
Comment (sevenzing) [reply stats/justfile:28]: you can put minus before command to ignore exit code and continue executing commands

Comment (sevenzing) [reply stats/justfile:28]: https://github.com/casey/just/tree/master?tab=readme-ov-file#ignoring-errors

**Pull request merged**